docs/source/workflow.rst

```diff
@@ -61,68 +61,13 @@ Deployment
-
-* ``--memory``: This is the value set in ``marathon.yaml`` as "mem".
-
-* ``--memory-swap``: Total memory limit (memory + swap). We set this to the same value
-  as "mem", rounded up to the nearest MB, to prevent containers being able to swap.
-
-* ``--net``: PaaSTA uses bridge mode to enable random port allocation.
-
-* ``--env``: Any environment variables specified in the ``env`` section will be here. Additional
-  ``PAASTA_``, ``MARATHON_``, and ``MESOS_`` environment variables will also be injected, see the
-  `related docs <yelpsoa_configs.html#env>`_ for more information.
-
-* ``--publish``: Mesos picks a random port on the host that maps to and exposes
-  port 8888 inside the container. This random port is announced to Smartstack
-  so that it can be used for load balancing.
+Kubernetes launches the Docker containers that comprise a PaaSTA service. Once a pod is scheduled to start, the kubelet on the node running the pod interacts with the container runtime
+through the Container Runtime Interface (CRI) to start the container defined in the pod specification.
 
-* ``--privileged``: Containers run by PaaSTA are not privileged.
-
-* ``--restart``: No restart policy is set on PaaSTA containers. Restarting
-  tasks is left as a job for the Framework (Marathon).
-
-* ``--rm``: Mesos containers are rm'd after they finish.
-
-* ``--tty``: Mesos containers are *not* given a tty.
-
-* ``--volume``: Volume mapping is controlled via the paasta_tools
-  configuration. PaaSTA uses the volumes declared in ``/etc/paasta/volumes.json``
-  as well as per-service volumes declared in ``extra_volumes`` declared
-  in the `soa-configs <yelpsoa_configs.html#marathon-clustername-yaml>`_.
-
-* ``--workdir``: Mesos containers are launched in a temporary "workspace"
-  directory on disk. Use the workdir sparingly and try not to output files.
-
-Mesos is the actual system that runs the docker images. In Mesos land these are
-called "TASKS". PaaSTA-configured tasks use exponential backoff to prevent
-unhealthy tasks from continuously filling up disks and logs -- the more times
-that your service has failed to start, the longer Mesos will wait before
-trying to start it again.
-
-Mesos *will* healthcheck the task based on the same healthcheck that SmartStack
-uses, in order to prune unhealthy tasks. This pruning is less aggressive than
-SmartStack's checking, so a dead task will go DOWN in SmartStack before it is
-reaped by Marathon. By default the healthchecks occur every 10 seconds, and a service
-must fail 30 times before that task is pruned and a new one is launched in its place.
-This means a task had 5 minutes by default to properly respond to its healthchecks.
+Note: Kubernetes support multiple container runtimes, including Docker (via "dockershim", which is deprecated and removed as of Kubernetes v1.24), containerd, and CRI-O.
+In Yelp, we use docker and are currently in the process of migrating to containerd.
```

Comment:
```suggestion
At Yelp, we use docker and are currently in the process of migrating to containerd.
```

Reply:
that said, this is almost done so we can probably leave this out (and remove the bit about k8s supporting multiple container runtimes too)